src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -37,6 +37,63 @@ trait OrientDbTestFixture extends BeforeAndAfterAll {
     def save(txn:RawlsTransaction)
   }
 
+  class EmptyWorkspace() extends TestData {
```

Comment:
I was going to use it in some of the Submission tests but I backed it out. IMO it's going to end up being useful at some point but I can remove it from this PR if you like.